app/repositories/company.js

```diff
@@ -1,5 +1,10 @@
+const db = require('../models');
 const Repository = require('./crudRepository');
 
-class CompanyRepository extends Repository {};
+class CompanyRepository extends Repository {
+  async getAll() {
+    return db.companies.findAll({ raw: true});
```

Comment:
Missing space before `}`.

Reply:
I think it is best to add this method here https://github.com/EdiOancea/ferret-api/blob/master/app/services/crudService.js